cross-media-measurement/.lintignore

```diff
@@ -0,0 +1,7 @@
+# Patterns to ignore when running linters.
+#
+# Each line is used a Git pathspec with the 'exclude' signature. Empty lines and those beginning with the '#' character are ignored.
+
+# Files which come from rbe_configs_gen and may have their own license.
+build/rbe/configs
```

Comment:
Oh. This is unfortunate. Could we just stop supporting RBE?

Reply:
The RBE config generator is how we generate the toolchain configs for Docker containers in general, as the autoconfig process won't work.

To clarify, it's just the Apache license and the non-generated files have license/copyright headers from the Bazel authors. The only file that's neither generated nor has its own header is empty.cc, which just contains an empty main function. Regardless, we don't want to format/lint third-party and/or generated code.